tests/utils.sh

```diff
@@ -304,10 +309,12 @@ function compare_command_sequence()
 
   while read -r f; do
     if [[ "${expected_res[$count]}" != "${f}" ]]; then
-      fail "line $line, statement $count: $msg
-Expected: \"${expected_res[$count]}\"
-but got:  \"${f}\"
-"
+      if ! fail &> /dev/null; then
+        printf '%bASSERT:%b line %s, statement %d: %s\n  %bExpected:%b %b\"%s\"%b\n  %b-> but got:%b %b\"%s\"%b\n' \
+          "${KW_COLOR_RED}" "${KW_COLOR_NONE}" "$line" "$count" "${msg}" \
```

Comment:
You don't need to use curly braces in these variables that have simple names, prefer not using them in this case.

Reply:
you're right. removed.